modules/op-release-notes-1-10.adoc

```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
 * `buckets`
 * `kube-api-qps`
 * `kube-api-burst`
-* `threads-per-controller`
\ No newline at end of file
+* `threads-per-controller`
+
+[id="release-notes-1-10-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.4
+
+With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-4_{context}"]
+=== Fixed issues
+
+* This update fixes the bundle resolver conversion issue for the `PipelineRef` field in a pipeline run. Now, the conversion feature sets the value of the `kind` field to `Pipeline` after conversion. 
```

Comment:
This sentence is correct grammatically but I would suggest that you modify this sentence in the "before this update and after this update" template that you have used in the rest of the content to maintain uniformity.